index.html

```diff
@@ -12,6 +12,14 @@
     <h1>Easy Lunch</h1>
     <div id="app"></div>
     <script type="text/javascript" src="index.js"></script>
+    <script type="text/javascript" src="restaurants.js"></script>
+    <script type="text/javascript" src="people.js"></script>
+    <script type="text/javascript">
+    const people = [jeremy, andrey, rahul];
+
+    console.log(randomRestaurantForToday(restaurants, people));
```

Comment:
Display the result on the page itself, not on console.log!

Reply:
Will do it in the next PR